More guidebook fixes

maland1 opened the pull request:
<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR
<!-- What did you change? -->
- Updating a number of guidebook pages to contain up to date information and references
- Slight adjustments for better flow or readability
- Fixing typos

## Why / Balance
<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. -->
Makes the Guidebook a bit more readable.

## Technical details
<!-- Summary of code changes for easier review. -->
XML Changes to guidebook pages

## Media
<!-- Attach media if the PR makes ingame changes (clothing, items, features, etc). 
Small fixes/refactors are exempt. Media may be used in SS14 progress reports with credit. -->


## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->


## Breaking changes
<!-- List any breaking changes, including namespaces, public class/method/field changes, prototype renames; and provide instructions for fixing them.
This will be posted in #codebase-changes. -->


**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->
:cl:
- tweak: Updating a number of guidebook pages to be more easily readable and accurate

maland1 committed:
First commit
- Resolving a few different pages, fixing typos and updating spellings and phrasing to be more coherent.

maland1 committed:
More slight changes

maland1 committed:
Reverting slightly the salvage page change

maland1 committed:
Fixing Science wording inconsistency

superjj18 commented:
I believe Nukie Corpsman is still referred to as Nukie Agent in the guide if you'd like to knock that out too

maland1 commented:
> I believe Nukie Corpsman is still referred to as Nukie Agent in the guide if you'd like to knock that out too

That is a good shout, I'll add that as well and take a quick look at the other nukie stuff to ensure it's up to date.

maland1 committed:
Few other small guidebook fixes
- Updating NukeOps page to properly refer to "Corpsman" rather than "Agent".
- Typo and slight wording change in Traitors.xml.
- Updating small typos in Antagonists.xml and Zombies.xml pages.

beck-thompson reviewed (approved):
as someone with dyslexia I probably shouldn't be reviewing this but it looks good! Thanks for the pr
